cli/stepfunctions/__init__.py

```diff
@@ -0,0 +1,224 @@
+        job_def = sequential_template.render(job_list=job_list)
+        entry = 'Pass_Job_{}'.format(jobs[0])
+    elif 'parallel' in config.options('order'):
+        job_txt = open(os.path.join(script_path, 'job_parallel.txt'), 'r').read()
+        parallel = config.get('order', 'parallel')
+        jobs = parallel.split(',')
+        jobs = map(lambda x: x.strip(), jobs)
+        job_list = []
+        for job in jobs:
+            sec = 10
+            section = 'job {}'.format(job)
+            if 'wait_time' in config.options(section):
+                sec = int(config.get(section, 'wait_time'))
+            if sec <= 0 or sec > 240:
+                print('wait_time must be between 1 and 240 seconds inclusive')
+                sys.exit(1)
+            handler = config.get(section, 'handler')
+            job_list.append({'name': job, 'sec': sec, 'handler': handler})
+        parallel_template = Template(job_txt)
+        job_def = parallel_template.render(job_list=job_list)
+        entry = 'Parallel_Job_Execution'
+    
+    # output dynamically generated template
+    new_cfn_txt = cfn_template.render(entry=entry, jobs=job_def)
+    open(os.path.join(tempdir, 'template.yaml'), 'w').write(new_cfn_txt)
+
+def _package(config_file, key_name, jobs_config):
+    script_path = os.path.dirname(os.path.realpath(__file__))
+    tempdir = _copy_source(script_path)
+    config = _edit_key_param(tempdir, config_file, key_name)
+    _package_jobs(tempdir, jobs_config, config, script_path)
+    _generate_template(script_path, tempdir, config)
+    return tempdir
+
+def _deeplink_url(region, stack_name):
+    # get outputs from cfn to use in url
+    try:
+        cloudformation = boto3.resource(
+            'cloudformation', region_name=region)
+        stack = cloudformation.Stack(stack_name)
+        stackId = stack.stack_id
+        outputs = stack.outputs
+        machineArn = filter(
+            lambda op: op['OutputKey'] == 'StateMachineArn', outputs
+        )[0]['OutputValue']
+    except ClientError as e:
+        print(e.response.get('Error').get('Message'))
+        sys.exit(1)
+
+    # fill and print url
+    url_region = '{}.'.format(region) if region != 'us-east-1' else ''
+    print('URL to Step Function State Machine:')
+    print('https://{}console.aws.amazon.com/states/home?region={}#/' \
+        'statemachines/view/{}?registered=true&stackId={}'.format(
+        url_region, region, machineArn, stackId))
+
+def deploy(args):
+    """Deploys the CloudFormation stack based on args
+
+    Args:
+        args: arguments passed in by argparse library
+    """
+    _check_docker()
+    _resolve_s3(args.bucket_name, args.region)
+    tempdir = _package(args.config_file, args.key_name, args.jobs_config)
+
+    # package and deploy the cloudformation stack
+    try:
+        path_dir = os.path.dirname(os.path.realpath(__file__))
+        path = os.path.join(path_dir, 'package.sh')
+        print(subprocess.check_output([path, tempdir, 
```

Comment:
Print something out like "Waiting for stack to create... " or "Packaging up all dependencies... this can take a moment"

Reply:
Sounds good!